tests/src/Reflection/EntityFieldsViaMagicReflectionExtensionTest.php

```diff
@@ -70,6 +70,11 @@ public function dataHasProperty(): \Generator
             'target_id',
             true,
         ];
+        yield 'field item list_interface: value' => [
+            \Drupal\Core\Field\FieldItemListInterface::class,
+            'target_id',
+            true,
+        ];
```

Comment:
Nit: test title is value but property is target_id

Reply:
Well, I copy pasted it from the test above it, so there is also the same nit there ;)

Changed them both then